client/source/class/cv/io/Client.js

```diff
@@ -90,6 +90,8 @@ qx.Class.define('cv.io.Client', {
     this.pass = '';
     this.device = '';
     this.headers = {};
+
+    this.delayedRestart = qx.util.Function.debounce(this.restart.bind(this), 50);
```

Comment:
Especially the client should be as much native JS and as little Qx as possible IMHO.
So is the use of this Qx helper function really necessary or could it be simply replaced by a `setTimeout` or similar?

Reply:
debounce !== setTimeout, I do not see any advantage of inventing the wheel twice when the framework you are using already provides a solution and you gain not a single percent less dependency by using it.
Everything is build and based on Qx so you already have 100% dependency, you can't get more dependency than that. Avoiding the usage of qx-stuff would only make sense if we had plans to remove qx at all. I do not have any plans for that, because that would be a enormous task with no real benefit.